src/System.Configuration/src/System.Configuration.csproj

```diff
@@ -189,7 +184,29 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <None Include="project.json" />
+    <Reference Include="System.Runtime" />
```

Comment:
I created a quick tool that sorts and trims the Reference items in the project for this ask, so I went ahead and ran it against the whole repo and created PR #15035 for it.